build.gradle

```diff
@@ -35,6 +35,7 @@ ext {
     compileSdkVersion = 28
     buildToolsVersion = '26.0.2'
     kotlinVersion = "1.3.61"
+    ndkVersion = '21.3.6528147'
```

Comment:
Why have you added this line?

Reply:
Actually github actions require ndk version to be specified for the check to run. It is a known issue, not fixing it causes the execution for `stripDebugDebugSymbols` fail which in turn fails entire process. Official Docs: [here](https://developer.android.com/studio/projects/install-ndk#apply-specific-version); known issue: [here](https://github.community/t/android-ndk-on-unix-is-incompatible-with-android-build-tools-3-6-0/17697)